LocalSettings.php

```diff
@@ -960,7 +960,15 @@
 	'wgWatchlistExpiry' => [
 		'default' => false,
 	],
-
+	
+	// HTTP
+	'wgHTTPConnectTimeout' => [
+		'default' => 3,
+	],
+	'$wgHTTPTimeout' => [
```

Comment:
Drop the ``$``.

Reply:
This has been fixed with https://github.com/miraheze/mw-config/pull/3799, which should be merged `first` to @RhinosF1's `patch` branch, then his PR to the `master` branch can be merged